package.json

```diff
@@ -9,17 +9,19 @@
     "fix": "yarn fix:js && yarn fix:other",
     "fix:other": "yarn prettier --write",
     "fix:js": "yarn test:js --fix",
-    "install": "yarn build",
+    "install": "yarn build && yarn zip",
     "prettier": "prettier \"**/*.{md,css,html,json,yml}\" --write",
     "start": "webpack --mode=development --watch ",
     "test": "yarn test:other && yarn test:js",
     "test:other": "yarn prettier --list-different",
-    "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.js\""
+    "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.js\"",
+    "zip": "cd extension && jszip-cli add . -i .DS_Store -o ../extension.zip -f"
```

Comment:
Probably it's safe to ignore this file by default in the lib.. :) (.DS_Store)

Reply:
Nope :) Someone may want to zip this file for testing purposes etc.